scholarx/2021/index.html

```diff
@@ -535,10 +535,10 @@ <h1 class="display-3 text-center mb-5">FAQ for Mentees</h1>
 </section>
 
 <section class="section past-section">
```

Comment:
`past-section` can also be removed because this is a separate section. So it needs to be changed as  `class="section"`